content/ita/careers.md

```diff
@@ -12,17 +12,22 @@ twitterlargeimage: "/images/careers/banner.png"
 
 ![Facile.it - lavora con noi!](/images/careers/banner.png)
 
-**Facile.it** è in costante crescita e siamo sempre alla ricerca di programmatori da aggiungere al nostro team. Questa proposta è rivolta a sviluppatori di grande esperienza, che credono nell’innovazione e sono sempre attenti alle nuove tecnologie e non vedono l’ora di metterle in pratica. Siamo un gruppo di lavoro dinamico, giovane e divertente. Siamo in totale quasi 80 programmatori e gestiamo numerosi progetti con differenti stack tecnologici.
+**Facile.it** is constantly growing and we are always looking for programmers to join our team.
+This proposal is aimed at highly experienced developers who believe in innovation and are always attentive to new technologies and can't wait to put them into practice. We are a dynamic, young and fun team.
+We are more than 100 programmers and manage numerous projects with different technology stacks.
```

Comment:
I would choose either developers or programmers and use the same term in the entire document.
Again, are we sure we want to talk about programmers/developers and not about other figures?

Reply:
Ditto
We can define them with generic 'experts' or we can list some of the roles, Do you have any suggestions?

Reply:
See comment above: we could choose the term "professionals".